src/core/grid_based_algorithms/lbgpu_cuda.cu

```diff
@@ -1408,11 +1408,12 @@ __device__ __inline__ float three_point_polynomial_larger_than_half(float u) {
 }
 
 template <std::size_t no_of_neighbours,
```

Comment:
Now `no_of_neighbours` no longer needs to be a template parameter, and you can rely on function overloads instead of enable_if